### PR TITLE
feat: Added Baby Ripple, Binance Coin & Monero

### PR DESCRIPTION
**Pull request by mlipovaca**
Please can you add this missing coins into library and release a new version, thanks 😄 

@marksuurland

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -37,13 +37,15 @@ npm install crypto-address-validator-ts
...
 * Axie Infinity/axs `'Axie Infinity'` or `'axs'`
+* Baby Ripple/babyxrp `'Baby Ripple'` or `'babyxrp'`
 * Badger DAO/badger `'Badger DAO'` or `'badger'`
 * Balancer/bal `'Balancer'` or `'bal'`
 * Bancor/bnt `'Bancor'` or `'bnt'`
 * Band Protocol/band `'Band Protocol'` or `'band'` only erc-20 validation no bep-20, xdai or fantom
 * Bankex/bkx `'Bankex'` or `'bkx'`
 * Basic Attention Token/bat `'Basic Attention Token'` or `'bat'`
 * BeaverCoin/bvc `'BeaverCoin'` or `'bvc'`
+* Binance Coin Mainnet/bnb `'Binance Coin Mannet'` or `'bnb'`
```
Comment (marksuurland) [inline README.md]: Small typo, mannet -> mainnet

File: tests/wallet_address_validator.ts [hunk 2]
```diff
@@ -294,6 +294,14 @@ describe('validate', function () {
             valid('rDTXLQ7ZKZVKz33zJbHjgVShjsBnqMBhmN', 'XRP', null);
         });
 
+        it('should return true for correct Baby Ripple addresses', function () {
```
Comment (marksuurland) [inline tests/wallet_address_validator.ts:306]: Please add a few tests for bnb as well (just for completeness)

**Review by marksuurland — APPROVED** [review]

**Comment by marksuurland** [comment]
> Please can you add this missing coins into library and release a new version, thanks 😄
> 
> @marksuurland

Done, some minor comments.
Thanks for your effort!

**Comment by mlipovaca** [comment]
> > Please can you add this missing coins into library and release a new version, thanks 😄
> > @marksuurland
> 
> Done, some minor comments. Thanks for your effort!

Hello, thanks for approval 😄  . I made the rest of the changes to the comments.

**Comment by marksuurland** [comment]
@mlipovaca currently on a small holiday (back on Sunday) will publish the new version then!

**Comment by PKChicks** [comment]
nobody uses Monero, or Binance coin. why was this added?

**Comment by marksuurland** [comment]
In this fork we are trying to support more coins, and I think bnb/monero are validate reasons (when it comes to market cap) to be added

**Comment by marksuurland** [comment]
Hi @mlipovaca saw there is an issue in the cnBase58.js you added:

var JSBigInt = require('./biginteger')['JSBigInt']; is missing.
I added 'big-integer' as npm dependency thinking it would do the trick, but it results in the tests failing / unstable results (xmr)
I pushed the changes to master (also fixed some tests), please have a look, once you created the changes/pr is merged I can create the new version on npm.

**Pull request merged**